lib/TestScriptRunnable.rb

```diff
@@ -56,9 +56,29 @@ def load_fixtures
 
       hash[fixture.id] = resource
       type = resource.resourceType
+      
+      # Need to consider where to put this code, such as a separate method before setup execition.
+      if fixture.autocreate
+        FHIR.logger.info "[.load_fixture] Autocreate Fixture: #{fixture.id}"
+        script.setup.action.each do |ac|
+            next if ac.operation == nil
+            if ac.operation.sourceId == fixture.id
+              FHIR.logger.warn "Possible duplication of fixture creation (autocreation and setup section)"
+              FHIR.logger.warn "Check setup in TestScript. Operation type: #{ac.operation.type.code} , sourceId: #{ac.operation.sourceId}"
+            end
+        end
 
-      script.setup.action.unshift action_create(fixture.id, type) if fixture.autocreate
-      script.teardown.action << action_delete(fixture.id, type) if fixture.autodelete
+        begin
+          client.create(get_resource_from_ref(fixture.resource))
```

Comment:
```suggestion
          client.create(resource)
```
The `get_resource_from_ref` has already been called and result stored in the resource variable.

Reply:
Will also need to update `id_map` using the id returned in the response of the create request. However, this `create` code is looking pretty similar to what we're doing in `execute`. Maybe we wrap each autocreate fixture in a simple operation and send it along to `execute`? What do you think about that?

Reply:
I agree but would vote to keep it as To-Do later this PR for a few reasons. I like abstracting codes and minimizing duplicates as much as it can, but in this case it's not many lines duplicate (and client.create is a very readable code). And I think the "execution" method in the engine covers "Setup, Test, Teardown Execution", but autocreate is outside the scope. But I'm not opposed to your idea overall. What do you think?

Reply:
Ah my bad, I meant `run`, not execute. So, it'd just be executing the create operation. But, we can definitely add it in outside of this PR.